src/DOM/WebStorage/Local.purs

```diff
@@ -0,0 +1,30 @@
+module DOM.WebStorage.Local
+( LocalStorage
+, getLocalStorage
+) where
+
+import DOM.WebStorage.Internal.Foreign
+
+import Control.Monad.Eff (Eff)
+import DOM (DOM)
+import Prelude ((<$>), (<<<))
+
+import DOM.WebStorage.String (class StrStorage)
+
+newtype LocalStorage = LocalStorage ForeignStorage
+
+instance strStorageLocalStorage :: StrStorage LocalStorage where
+  length = length <<< unLocalStorage
+  key = key <<< unLocalStorage
+  getItem = getItem <<< unLocalStorage
+  setItem = setItem <<< unLocalStorage
+  removeItem = removeItem <<< unLocalStorage
+  clear = clear <<< unLocalStorage
+
```

Comment:
So PS does not allow instances on foreign types? Ok, but still sessionStorage and localStorage could still share the same StrStorage instance -right? and therefore could share the same newtype around ForeignStorage? Same for MockStorage actually, as you made the JavaScript API a ForeignStorage too there. They are all the same type, I don't really get why you need to make them different with newtypes. What do I miss?